core/src/test/java/com/yahoo/imapnio/async/response/ImapResponseMapperTest.java

```diff
@@ -1299,4 +1301,123 @@ public void testParseToSearchResultNotOK() throws IOException, ProtocolException
         Assert.assertNotNull(actual, "ImapAsyncClientException should occur.");
         Assert.assertEquals(actual.getFailureType(), FailureType.INVALID_INPUT, "Failure type mismatched.");
     }
+
+    /**
+     * Tests parseToEnableResult method successfully.
+     *
+     * @throws IOException will not throw
+     * @throws ProtocolException will not throw
+     * @throws ImapAsyncClientException will not throw
+     */
+    @Test
+    public void testParseToEnableResult() throws IOException, ProtocolException, ImapAsyncClientException {
+        final ImapResponseMapper mapper = new ImapResponseMapper();
+        final List<IMAPResponse> content = new ArrayList<>();
+        content.add(new IMAPResponse("* some junks\r\n"));
+        content.add(new IMAPResponse("* ENABLED CONDSTORE\r\n"));
```

Comment:
Make it one line 
* ENABLED CONDSTORE QRSYNC\r\n

Reply:
done